File: Poseidon_SpringAPI/src/main/java/fr/orionbs/models/Rating.java
package fr.orionbs.models;

import lombok.*;

import javax.persistence.*;

@AllArgsConstructor
@NoArgsConstructor
@Builder
@Getter
@Setter
@Entity(name = "rating")
public class Rating {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Integer id;
    private String moodysRating;
    private String sAndPRating;
    private String fitchRating;
    private Integer orderNumber;

    @Override
    public String toString() {
        return "Rating{" +
                "id=" + id +
                ", moodysRating='" + moodysRating + '\'' +
                ", sAndPRating='" + sAndPRating + '\'' +
                ", fitchRating='" + fitchRating + '\'' +
                ", orderNumber=" + orderNumber +
                '}';
    }
}

File: Poseidon-AngularFront/src/app/rating/rating.ts
export class Rating {
	id?: number;
	moodysRating?: string;
	sandPRating?: string;
	fitchRating?: string;
    orderNumber?: number;
	
	constructor(moodysRating?: string, sandPRating?: string, fitchRating?: string, orderNumber?: number) {
		this.moodysRating = moodysRating;
		this.sandPRating = sandPRating;
		this.fitchRating = fitchRating;
		this.orderNumber = orderNumber;
	}

}
